dashboard/app/models/ability.rb

```diff
@@ -126,13 +126,20 @@ def initialize(user)
         CodeReviewComment.user_can_review_project?(project_owner, user, project_id)
       end
 
-      can :create, CodeReview do |code_review, project_owner_id|
+      can :create, CodeReview do |code_review, project|
         code_review.user_id == user.id &&
-        project_owner_id == user.id
+        project.owner_id == user.id
       end
       can :edit, CodeReview, user_id: user.id
-      # TODO: teachers and peers should also be able to see the code review
-      can :read, CodeReview, user_id: user.id
+      can :index_code_reviews, Project do |project|
+        # The user can see the code review if one of the following is true:
+        # 1) the user is the project owner
+        # 2) the user is the teacher of the project owner
+        # 3) the user and the project owner are in the same code reivew group
```

Comment:
Another criteria is that the teacher has turned on code review for the section, do we want to include that here?

Reply:
Thanks for catching this.  From our meeting with Mike, I think we want to add the check for if code review is turned on to case 3 above.  I'll pick this up in a future PR.